### DISCUSSION TITLE
Conditional Variables

### DISCUSSION THREAD

**Discussion by vitt95** [YetiForceCompany/YetiForceCRM#15984] [How to] [answered]
Hi, is there a way to conditionally display variables into pdf markup ?

Example: 

`$(record: name )$ ? <td>$(record: name)</td> : 'empty space'.`

The core question is if is possible to display html conditionally also. Thank you.

**Comment by vitt95** [comment, marked as answer]
Okay I'll answer myself for other dev who's facing this.  After spent a couple of hours wondering if was possible , i've implemented a custom TextParser class who checks for fields availability and returns formatted html code as well. This is supported by Yetiforce framework natively.

**Comment by mariuszkrzaczkowski** [comment]
TextParser in PDf supports the contribution from the twig library (https://twig.symfony.com/doc/3.x/tags/if.html), there you can create different conditions
